dev/update-output-repos.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const execa = require('execa');
+const tmp = require('tmp');
+tmp.setGracefulCleanup();
+
+const currentVersion = require('../package').version;
+const EMBER_PATH = require.resolve('../bin/ember');
+const isStable = !currentVersion.includes('-beta');
```

Comment:
should  we use the  semver package to parse for the actual version extension thingy? just in case we want to publish alphas eventually 😅

Reply:
Ya, good point, will update.